### PR TITLE
feat: add prop alwaysScrolled to header

### PR DESCRIPTION
**Pull request by mato-a**
# Add property alwaysScrolled to header

If this prop is set to true header will be always thin even if the page was not scrolled down.

Class `header--sticky` will be added always to the header.

This is useful for example in fullscreen mode where I'd like to make the navbar as small as possible.

### CODE REVIEW

**Comment by nowseemee** [comment]
Thanks for your PR! 🎉 🎉 🎉 
I'll discuss this with the team and come back to you soon.

File: packages/components/src/components/telekom/app-header/app-header.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ export class Header {
   @Prop() addonNavigation?: any = [];
   @Prop() activeRouteId: string;
   @Prop() activeSectorId?: string;
+  @Prop() alwaysScrolled?: boolean = false;
```
Comment (nowseemee) [inline packages/components/src/components/telekom/app-header/app-header.tsx:60]: can you rename the new prop to `sticky` please?

**Review by nowseemee — CHANGES_REQUESTED** [review]
Thanks a lot for your PR! I'd like you to change the name of the prop to `sticky` and also add this to `app-shell` and pass it down to `app-header`.

**Comment by nowseemee** [comment]
Thanks for adding your changes. Can you please run `yarn build` in the `components` package and commit the changes? Many thanks for all your contributions!

**Comment by nowseemee** [comment]
thanks a lot @mato-a - i'll publish this to NPM today and let you know ✌🏻

**Comment by mato-a** [comment]
@nowseemee great 👍 

Thanks for guidance 😉

**Comment by nowseemee** [comment]
@mato-a, it's released: `3.0.0-beta.33`

**Pull request merged**